torch/nn/_intrinsic/qat/modules/conv_bn_relu.py

```diff
@@ -0,0 +1,227 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import math
+import torch
+from torch.nn.modules.conv import _ConvNdBase
+from torch.nn.modules.utils import _pair
+from torch.nn import init
+from torch.nn._intrinsic import ConvBn2d as NNConvBn2d
+from torch.nn._intrinsic import ConvBnReLU2d as NNConvBnReLU2d
+from torch.nn import Parameter
+import torch.nn.functional as F
+
+
+class ConvBn2d(_ConvNdBase):
```

Comment:
Similar comment: Might be  better to inherit from nn.Conv2d

Reply:
see previous comment